clouddriver-kubernetes/src/test/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/KubernetesNamedAccountCredentialsInitializerSpec.groovy

```diff
@@ -117,7 +117,6 @@ class KubernetesNamedAccountCredentialsInitializerSpec extends Specification {
     1 * accountCredentialsRepository.save("test-account", _ as KubernetesNamedAccountCredentials) >> { _, creds ->
       credentials = creds
     }
-    1 * namerRegistry.getNamingStrategy("kubernetesAnnotations") >> Mock(KubernetesManifestNamer)
```

Comment:
This used to run for both v1 and v2, even though it was only needed for v2.  After the refactor it only runs for v2.